api/debug.go

```diff
@@ -95,3 +130,241 @@ func (d *DebugAPI) TraceBlockByHash(
 
 	return results, nil
 }
+
+func (d *DebugAPI) TraceCall(
+	ctx context.Context,
+	args TransactionArgs,
+	blockNrOrHash rpc.BlockNumberOrHash,
+	config *tracers.TraceCallConfig,
+) (interface{}, error) {
+	if err := rateLimit(ctx, d.ratelimiter, d.logger); err != nil {
+		return nil, err
+	}
+
+	txEncoded, err := encodeTxFromArgs(args)
+	if err != nil {
+		return nil, err
+	}
+
+	// Default address in case user does not provide one
+	from := d.config.Coinbase
+	if args.From != nil {
+		from = *args.From
+	}
```

Comment:
Good catch :ok_hand: I checked the Geth codebase, and they simply default to the zero address: https://github.com/onflow/go-ethereum/blob/master/internal/ethapi/transaction_args.go#L79-L85, instead of reverting.
I've updated the endpoint to match this functionality, in https://github.com/onflow/flow-evm-gateway/pull/605/commits/93d1d3ead1b750c57ce5d9e184f75fc7779f53c7.